SAMLPortal/Startup.cs

```diff
@@ -67,25 +72,25 @@ public static void Configure(IApplicationBuilder app, IWebHostEnvironment env, S
 			}
 			else
 			{
-				app.UseExceptionHandler("/Home/Error");
-				context.Database.Migrate();
+				app.UseExceptionHandler("/Error");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Refactor this method to add validation of parameter 'app' before using it.](https://app.codacy.com/manual/zegorax/SAMLPortal/pullRequest?prid=5343548)